packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -12,13 +12,27 @@
         </p>
 
         <f-button
-            :class="$style['c-checkout-redirectButton']"
+            :class="$style['c-checkout-errorDialogButton']"
+            :data-gtm="isDuplicateOrderError ? 'engagement|dialog_duplicate_order_warning|click_acknowledge' : undefined"
             button-size="large"
+            :button-type="isDuplicateOrderError ? 'secondary' : 'primary'"
             data-test-id="redirect-to-menu-button"
```

Comment:
Not related to your pr but this test id may be wrong. The close method redirects only in some instances.